msticpy/nbtools/process_tree.py

```diff
@@ -0,0 +1,430 @@
+    b_plot.xgrid.grid_line_color = "navy"
+    b_plot.xgrid.grid_line_alpha = 0.1
+    b_plot.axis.major_label_standoff = 0
+
+
+def _get_tool_tips(schema: ProcSchema):
+    """Return tool tip formatter."""
+    return [
+        ("Process", f"@{schema.process_name}"),
+        ("PID", "@PID"),
+        ("CmdLine", f"@{schema.cmd_line}"),
+        ("SubjUser", f"@{schema.user_name}"),
+        ("SubjLgnId", f"@{schema.logon_id}"),
+        ("TargLgnId", f"@{schema.target_logon_id}"),
+        ("Time", f"@{schema.time_stamp}{{%F %T}}"),
+    ]
+
+
+def _create_js_callback(source: ColumnDataSource, result_var: str) -> CustomJS:
+    """Create and return CustomJS callback to set Python variable."""
+    ret_var_js = """
+        // get data source from Callback args
+        var inds = source.selected.indices;
+        var output = [];
+        for (var i = 0; i < inds.length; i++) {
+            output.push(source.data[itemkey][inds[i]])
+        }
+        out_str = JSON.stringify(output);
+        py_str = `${output_var} = ${out_str}`;
+        console.log(py_str);
+        IPython.notebook.kernel.execute(py_str);
+    """
+    get_selected = CustomJS(
+        args=dict(source=source, itemkey="proc_key", output_var=result_var),
+        code=ret_var_js,
+    )
+    return get_selected
+
+
+def _create_fill_map(
+    source: ColumnDataSource, source_column: str = None
+) -> Tuple[Union[factor_cmap, linear_cmap], Optional[ColorBar]]:
+    """Create factor map or linear map based on `source_column`."""
+    fill_map = "navy"
+    color_bar = None
+    if source_column is None or source_column not in source.data:
+        return fill_map, color_bar
+
+    col_kind = source.data[source_column].dtype.kind
+    if col_kind in ["b", "O"]:
+        s_values = set(source.data[source_column])
+        if np.nan in s_values:
+            s_values.remove(np.nan)
+        values = list(s_values)
+        fill_map = factor_cmap(
+            source_column, palette=viridis(max(3, len(values))), factors=values
+        )
+    elif col_kind in ["i", "u", "f", "M"]:
+        values = [val for val in source.data[source_column] if not np.isnan(val)]
+        fill_map = linear_cmap(
+            field_name=source_column,
+            palette=viridis(256),
+            low=np.min(values),
+            high=np.max(values),
+        )
+        color_bar = ColorBar(
+            color_mapper=fill_map["transform"], width=8, location=(0, 0)  # type: ignore
+        )
+    return fill_map, color_bar
+
+
+# pylint: disable=too-many-arguments
+def _create_vert_range_tool(
+    data, min_y, max_y, plot_range, width, height, x_col, y_col, fill_map="navy"
+):
+    """Return vertical range too for plot."""
+    rng_select = figure(
+        plot_width=width, plot_height=height, y_range=(min_y - 1, max_y + 1)
+    )
+
+    x_dodge = dodge(x_col, -0.5)
+    rng_select.rect(
+        x=x_dodge,
+        y=y_col,
+        width=1.2,
+        height=0.8,
```

Comment:
I am finding this too small to be usable if you have more than a small set of data - suggest upping it to 1.2

Reply:
Pretty sure this is the glyph height so don't see how this affects the range box height